GameData/NearFutureSpacecraft/Localization/ru.cfg

```diff
@@ -0,0 +1,150 @@
+﻿// Draft russian localization. No tags.
+// © Dr. Jet
+
+// As tags mostly mimic Squad ones - it may be wize to remove them from localization files and write part config lines like this: 
+// tags = #autoLOC_500291 nearfuture orbital
```

Comment:
That's an absolutely hideous amount of work to go back and do that for everything. If you want to volunteer and validate though, be my guest